docs/node-implementation/blocks.md

```diff
@@ -0,0 +1,15 @@
+title: Node Implementation - Blocks
+description: Review of block handling in the current node implementation
+
+# Node Implementation - Blocks
+
+--8<-- "wip-living-whitepaper.md"
+
+## Block publishing and propagation
+
```

Comment:
```suggestion
Nodes use a modified gossip protocol for message distribution that enables quick distribution of blocks and votes across the network while distributing the load required to propagate messages across multiple nodes rather than each node having to respond to requests from every other node.  Prioritization of messages are focused on the Principal Representative nodes that make up the core consensus mechanism where they receive blocks and votes directly from other nodes and then messages are spread to the rest of the network via gossiping.

```